src/views/auth/LoginPage.vue

```diff
@@ -0,0 +1,213 @@
+<template>
+  <form class="login-container" @submit.prevent="submitForm">
+    <h1>Login</h1>
+    <div class="input-layout">
+      <div class="form__group field">
+        <input
+          class="form__field"
+          placeholder="email"
+          name="email"
+          id="email"
+          v-model="account.email"
+        />
+        <label for="Email" class="form__label">Email</label>
+        <span v-for="error in v$.email.$errors" :key="error.$uid">
+          {{ error.$message }}
+        </span>
+      </div>
+      <div class="form__group field">
+        <input
+          type="password"
+          class="form__field"
+          placeholder="password"
+          name="password"
+          id="password"
+          v-model="account.password"
+        />
+        <label for="password" class="form__label">Password</label>
+        <span v-for="error in v$.password.$errors" :key="error.$uid">
+          {{ error.$message }}
+        </span>
+      </div>
+    </div>
+    <p v-if="checkLogin">Email or Password incorrect, Please enter again!</p>
+    <Button type="submit" label="Login" />
+  </form>
+</template>
+
+<script lang="ts" setup>
+import { computed, reactive, ref } from "vue";
+import useVuelidate from "@vuelidate/core";
+import { required, minLength, email } from "@vuelidate/validators";
+import axios from "axios";
+import { useRouter } from "vue-router";
+import Button from "primevue/button";
+import https from "@/services/axios";
+import { httpStatus } from "@/services/enumStatus";
+const router = useRouter();
+const checkLogin = ref();
+const account = reactive({
+  email: "",
+  password: "",
+});
+const rules = computed(() => {
+  return {
+    email: { required, minLength: minLength(10), email },
+    password: { required, minLength: minLength(8) },
+  };
+});
+
+const v$ = useVuelidate(rules, account);
+
+const submitForm = async () => {
+  const result = await v$.value.$validate();
+
+  if (result) {
+    const fetchAccount = async () => {
+      axios
+        .post(`${https.httpsLogin.defaults.baseURL}/auth/login`, {

```

Comment:
dùng axiosService là được rồi e
```suggestion

       axiosService.post('/auth/login', {

```